tasks/35_array-leaders/array-leaders.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * arrayLeaders - finds all the LEADERS in the array
+ *
+ * @param {Array<Number>} numbers
+ * @return {Array<Number>}
+ */
+export function arrayLeaders(numbers) {
+  const getSum = (arr) => {
```

Comment:
Is there any reason to declare getSum function inside arrayLeaders function?

Reply:
Fixed